pyproject.toml

```
@@ -26,3 +26,14 @@ skip-string-normalization = true
 [build-system]
 requires = ['setuptools==63.4.3']
 build-backend = "setuptools.build_meta"
+[tool.isort]
+add_imports = ["from __future__ import annotations"]
+append_only = true
+line_length = 110
+combine_as_imports = true
+default_section = "THIRDPARTY"
+known_first_party = "airflow,airflow_breeze,docker_tests,docs,kubernetes_tests,tests"
```

```suggestion
known_first_party = ["airflow", "airflow_breeze", "docker_tests", "docs", "kubernetes_tests", "tests"]
```

toml has lists so we should use them